### ISSUE TITLE
Viewing an extention in settings menu breaks atleast that extention

### ISSUE THREAD

**Issue by Chattaeus** [AccelerateNetworks/an-mobile-ios#1]
Two features/issues that need fixing with the iOS app.
1: Logs should be enabled by default. Trust me we'll need that.
2: The big thing now though is that whenever you click on the extension in the menu it breaks the config. Tested this quite a few times and it happened every one. You have to delete the account and then re-provision.

**Comment by piajesse** [comment]
First of all that is 2 separate issues, this thread should focus on the menu issue only
after some digging in the logs, I've noticed the following so far. will dive into the code to see if I can figure out why this happens
before:
```
From: "piajesse-Stock" <sips:333@acceleratenetworks.sip.callpipe.com>;tag=y6NlfZ0x7
To: "piajesse-Stock" <sips:333@acceleratenetworks.sip.callpipe.com>
```

after:
```
From: "piajesse-Stock" <sip:333@acceleratenetworks.sip.callpipe.com>;tag=U0YPD6VV0
To: "piajesse-Stock" <sip:333@acceleratenetworks.sip.callpipe.com>
```

**Comment by piajesse** [comment]
[https://github.com/AccelerateNetworks/an-mobile-ios/commit/9b77cea175603a8b26c2611e3a329c946f2e064d](url)
looks like this issue was patched and not tested then reverted, let me try the patch

**Comment by piajesse** [comment]
that patch fixed the issue, pushing to github and testflight now, please test @Chattaeus

**Comment by piajesse** [comment]
This is confirmed to be fixed with latest update

**Comment by piajesse** [comment]
this is still a problem, right now if you visit the extentions settings it clears the port, and sets it to the default `5060`. this issue actually can fix AccelerateNetworks/an-mobile-ios#2

**Comment by piajesse** [comment]
this was a problem with our provisioning configs, this issue is fixed on the new config generator
